### PR TITLE
Feature/replace logging

### PR DESCRIPTION
**Pull request by DavidSubiros**
### What
- Replaced logging library with dp-logger in order to comply with logging standards
- format logs according to standards (with separate fields for variables, implemented in new LogEvent class)
- Added `GET /health` endpoint

### How to review

- Make sure code changes make sense
- Make sure unit tests pass
- You can run `make debug` and validate that the log format is correct.

### Who can review

Anyone

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,6 +8,14 @@
...
 
     <repositories>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>central</id>
+            <name>Central Repository</name>
+            <url>https://repo.maven.apache.org/maven2</url>
+        </repository>
```
Comment (CarlHembrough) [inline pom.xml:18]: I don't think we need to add an additional repository as the `dp-logging` library is hosted by jitpack?

File: src/main/java/dp/healthcheck/Health.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+@RestController
+public class Health {
+
+    @RequestMapping("/health")
+    HttpStatus healthCheck() {
+        return HttpStatus.OK;
+    }
+
+}
```
Comment (CarlHembrough) [inline src/main/java/dp/healthcheck/Health.java:16]: Are we going to migrate to use this endpoint as part of this task?

**Review by CarlHembrough — CHANGES_REQUESTED** [review]
Changes look good - just two minor queries

File: src/main/java/dp/healthcheck/Health.java (hunk 2 above)
Comment (DavidSubiros) [reply src/main/java/dp/healthcheck/Health.java:16]: This wasn't part of the task, I can remove it if required. But I did it for consistency with other services. Or shall I create a separate PR?

File: pom.xml (hunk 1 above)
Comment (DavidSubiros) [reply pom.xml:18]: ok, I will try to build and run the service without Central Repository

Comment (DavidSubiros) [reply pom.xml]: validated and pushed a new commit without this dependency

**Review by CarlHembrough — APPROVED** [review]
Good work 👍

**Pull request merged**